libs/react-components/tedi/src/components/tag/tag.tsx

```diff
@@ -0,0 +1,64 @@
+import cn from 'classnames';
+import { MouseEventHandler } from 'react';
+
+import { BreakpointSupport, useBreakpointProps } from '../../helpers';
+import ClosingButton from '../closing-button/closing-button';
+import { Icon } from '../icon/icon';
+import { Spinner } from '../spinner/spinner';
+import styles from './tag.module.scss';
+
+type TagColor = 'primary' | 'secondary' | 'danger';
+
+type TagBreakpointProps = {
+  /**
+   * Function to be called when the close button is clicked.
+   * If provided, a close button will be rendered inside the Tag.
+   */
+  onClose?: MouseEventHandler<HTMLButtonElement>;
```

Comment:
Doesn't need to be in breakpointProps, no need to call different onClose trigger on different breakpoints